manifests/cluster-infra/5a_amgh_deployment.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: amgh
```

Comment:
amgh is a good name for the cli, but I am wondering since this is refered to in so many place whether we should put something more descriptive like - `githubNotifier` . Still short but indicates its purpose a bit better.

Reply:
okay, I'll make the change to the name. but I think having `am` in the name will be more descriptive as it's primarily a webhook receiver for the alertmanager. what about `amghNotifier` or even `amgithubNotifier`?

Reply:
amGithubNotifier